packages/core/test/lib/commands/compile.js

```diff
@@ -100,7 +100,9 @@ describe("compile", function () {
     });
 
     it("prints a list of docker tags", async function () {
-      this.timeout(20000);
+      this.timeout(
+        60000 * 2  // oh boy!
+      );
```

Comment:
2 minutes is an egregious limit for a test. OTOH, these tests shouldn't hit the network. I opened #5639 to capture that.